timesketch/lib/analyzers/interface.py

```diff
@@ -1150,3 +1165,196 @@ def get_kwargs(cls):
     def run(self):
         """Entry point for the analyzer."""
         raise NotImplementedError
+
+
+class AnalyzerOutputException(Exception):
+    """Analyzer output exception."""
+
+
+class AnalyzerOutput:
+    """A class to record timesketch analyzer output.
+
+    Attributes:
+        platform (str): [Required] Analyzer platfrom.
+        analyzer_identifier (str): [Required] Unique analyzer identifier.
+        analyzer_name (str): [Required] Analyzer display name.
+        result_status (str): [Required] Analyzer result status.
+            Valid values are success, no-findings, error.
```

Comment:
`no-findings` is a bit ambiguous IMO. Is it because of an error or something else? Can we skip this status and only have `success` and `error`? Then to know if there are no findings you can check the result attributes.

Reply:
Yes, that should work as well.